Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py

```diff
@@ -1189,6 +1473,158 @@ def fetch_security_issue_command(client: Client, args: Dict[str, Any]):
     )
 
 
+def list_sysadmin_assets_command(client: Client, args: Dict[str, Any]):
+    assets = client.get_sysadmin_assets(args)
+
+    if len(assets) == 0:
+        raise DemistoException("No Sysadmin assets found")
+
+    # minimal readable table: id, hostname, last_communication, category
+    readable_headers = ["id", "hostname", "last_communication", "category"]
+    readable_assets = [
+        {
+            "id": str(a.get("id")),
+            "hostname": a.get("hostname"),
+            "last_communication": iso8601_to_human(a.get("last_communication")),
+            "category": a.get("category"),
+        }
+        for a in assets
+    ]
+
+    return CommandResults(
+        outputs=createContext(assets, removeNull=True),
+        outputs_prefix="Cyberwatch.SysadminAsset",
+        raw_response=assets,
+        outputs_key_field="id",
+        readable_output=tableToMarkdown(
+            "Cyberwatch Sysadmin Assets",
+            readable_assets,
+            readable_headers,
+            removeNull=False,
+            is_auto_json_transform=True,
+            date_fields=["last_communication"],
+        ),
+    )
+
+
+def fetch_sysadmin_asset_command(client: Client, args: Dict[str, Any]):
+    asset = client.get_sysadmin_one_asset(args)
+
+    if len(asset) == 0:
+        raise DemistoException("Sysadmin asset not found")
+
+    readable_headers = [
+        "id",
+        "hostname",
+        "description",
+        "last_communication",
+        "category",
+        "deploying_period_id",
+        "rebooting_period_id",
+        "policy_id",
+        "ignoring_policy_id",
+    ]
+    readable_asset = {
+        "id": str(asset.get("id")),
+        "hostname": asset.get("hostname"),
+        "description": str(asset.get("description")),
+        "last_communication": iso8601_to_human(asset.get("last_communication")),
+        "category": str(asset.get("category")),
+        "deploying_period_id": str(asset.get("deploying_period_id")),
+        "rebooting_period_id": str(asset.get("rebooting_period_id")),
+        "policy_id": str(asset.get("policy_id")),
+        "ignoring_policy_id": str(asset.get("ignoring_policy_id")),
+    }
+
+    return CommandResults(
+        outputs=createContext(asset, removeNull=False),
+        outputs_prefix="Cyberwatch.SysadminAsset",
+        raw_response=asset,
+        outputs_key_field="id",
+        readable_output=tableToMarkdown(
+            "Cyberwatch Sysadmin Asset",
+            readable_asset,
+            readable_headers,
+            removeNull=False,
+            is_auto_json_transform=True,
+            date_fields=["last_communication"],
+        ),
+    )
+
+
+def list_compliance_assets_command(client: Client, args: Dict[str, Any]):
+    assets = client.get_compliance_assets(args)
+
+    if len(assets) == 0:
+        raise DemistoException("No Compliance assets found")
```

Comment:
it should not be an exception, it's a valid response, please send it asa demisto results not exception